src/Compilers/Test/Core/Compilation/TestOperationVisitor.cs

```diff
@@ -735,6 +735,10 @@ public override void VisitUnaryOperator(IUnaryOperation operation)
 
             AssertConstrainedToType(operatorMethod, operation.ConstrainedToType);
             Assert.Same(operation.Operand, operation.ChildOperations.Single());
+
+            // Directly get the symbol for this operator from the semantic model.  This allows us to exercise
+            // potentially creating synthesized intrinsic operators.
+            operation.SemanticModel?.GetSymbolInfo(operation.Syntax);
```

Comment:
>operation.SemanticModel?.GetSymbolInfo(operation.Syntax); [](http://example.com/codeflow?start=12&length=57)

I though the intent was to not just get the symbol, but also recreate it if it is built-in.